src/boot/notify.js

```diff
@@ -9,48 +9,47 @@ const closeBtn = '✖';
  * Usage for notify plugin
  *
  * Add to quasar.conf.js
- *  plugins: ['notify]
+ *  plugins: ['notify']
  *
  * @example
  * this.$notify.success('successfully-created');
+ * 
+ * @notes
+ * This will only utilize $t
+ * You will need to translate manually if it is interpolated
```

Comment:
@wilbertverayin Could you provide an example for this?

Reply:
i have added example usage